hackathon_site/hardware/views.py

```diff
@@ -46,11 +47,13 @@ def get(self, request, *args, **kwargs):
         return self.retrieve(request, *args, **kwargs)
 
 
-class IncidentListView(mixins.ListModelMixin, generics.GenericAPIView):
+class IncidentListView(
+    mixins.ListModelMixin, mixins.CreateModelMixin, generics.GenericAPIView
+):
     queryset = Incident.objects.all().select_related(
         "order_item", "order_item__order__team", "order_item__hardware"
     )
-    serializer_class = IncidentSerializer
+    serializer_class = IncidentListSerializer
```

Comment:
do you need this if you have the get_serializer_class() method? I think that method overwrites serializer_class property

Reply:
hmm actually, nvm keep it, it defaults to that serializer if we're not making a get or post request

Reply:
I'd probably agree, you can remove it if you have a `get_serializer_class` method. And if there's no method on the view for the other HTTP verbs, they API will give a 405 Method Not Allowed anyway.